services/common/workflow.js

```diff
@@ -14,6 +14,8 @@ export const ONEMAC_TYPE = {
   WAIVER_RENEWAL_WITHDRAW: "waiverrenewalwithdraw",
   WAIVER_RAI: "waiverrai",
   WAIVER_EXTENSION: "waiverextension",
+  WAIVER_EXTENSION_B: "waiverextensionb",
+  WAIVER_EXTENSION_C: "waiverextensionc",
```

Comment:
I like that you kept the plain "waiverextension" type as well, since we still have those leftover packages without a b or c...